com.microsoft.mrtk.input/Utilities/PoseSource/HandJointPoseSource.cs

```diff
@@ -13,6 +13,17 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public class HandJointPoseSource : HandBasedPoseSource
     {
+        /// <summary>
+        /// Empty constructor for inspector instantiation
+        /// </summary>
+        public HandJointPoseSource() { }
+
+        public HandJointPoseSource(Handedness hand, TrackedHandJoint joint)
```

Comment:
I'm not sure I see this new constructor being used anywhere. Do we still want it?

Reply:
Good call